app/src/main/java/io/neurolab/model/FeedbackSettings.java

```diff
@@ -0,0 +1,64 @@
+package io.neurolab.model;
+
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.annotation.Nullable;
+import android.support.v4.app.FragmentActivity;
+import android.widget.TextView;
+
+import io.neurolab.R;
+
+public class FeedbackSettings extends FragmentActivity implements SharedPreferences.OnSharedPreferenceChangeListener {
```

Comment:
Usually a `model` means something different than a fragment. Let's change the package to `activities -> settings`